src/yuzu/discord_impl.cpp

```diff
@@ -29,23 +30,82 @@ void DiscordImpl::Pause() {
     Discord_ClearPresence();
 }
 
+std::string GetGameString(const std::string& title) {
```

Comment:
```suggestion
static std::string GetGameString(const std::string& title) {
```

since it's only used in this file.